### PR TITLE
Rudest

### PR DESCRIPTION
**Pull request by mnguyen36**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import {Router} from 'express';
+import http from 'http';
+
+import parsePayload from '../lib/parser'
+
+const app = express();
+app.server = http.createServer(app);
+const router = Router();
+
+router.get('/', async (req, res) => {
```
Comment (mnguyen36) [inline lib/index.js]: I guess I could pull this out into it's own router.js but u do it

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    try {
+        res.send(await parsePayload(req.query.data));
+    } catch (err) {
+        res.send('fkedup');
```
Comment (mnguyen36) [inline lib/index.js]: I guess I can't have this here either

File: lib/index.js (hunk 1 above)
Comment (gempesaw) [reply lib/index.js]: too lazy

unless you wanna test that you mapped `/` to this path

File: lib/index.js (hunk 2 above)
Comment (gempesaw) [reply lib/index.js]: @daeious

**Pull request merged**